### ISSUE TITLE
Why the stack trace and why the log output on 404?

### ISSUE THREAD

**Issue by mlarsson** [pleerock/routing-controllers#295]
When throwing an NotFoundError as described in the documentation I would expect that my user would get the following back:  

404 with the body:
```
{
    "name": "NotFoundError",
    "message": "My error text"
}
```

At least this is how I read the documentation. But I also get an attribute "stack" in this message. Provideing my users with A) information I don't want to share, B) useless information (from their perspective). How do I get rid of this unwanted/undocumented attribute?

I also see a stack trace in the servers log. Telling me that a NotFoundError occurred. How do I remove that?

Thank you for your help.

Kind regards Morten

**Comment by 19majkel94** [comment]
```ts
if (error.stack && this.developmentMode)
    processedError.stack = error.stack;
```

This is default handler code. So change your `NODE_ENV` to `production` or provide `developmentMode` option in routing-controllers config.

**Comment by mlarsson** [comment]
Cool thank you. Actually upgrading from 0.7.0 to 0.7.2 fixed my issues. I have another one now though. I get html content back on a 404 instead of json. Do you have quick fix for that as well? Note: all my controllers are already annotated with "@JsonController".

**Comment by 19majkel94** [comment]
It was fixed in #277, waiting for new release on npm 😞 

I added a test case to be sure:
https://github.com/pleerock/routing-controllers/commit/17a53c60ddd169b57d66287eedac79fbada06ea5

**Comment by pleerock** [comment]
This will be released today in 0.7.3
